demo/index.tsx

```diff
@@ -1,31 +0,0 @@
-import React from 'react';
```

Comment:
Does this need to be kept?

Reply:
It looks like it's not needed. Though I think we need to update the eslint rules.